cranelift/isle/isle/src/sema.rs

```diff
@@ -385,6 +389,24 @@ impl Term {
                     infallible: *infallible,
                 })
             }
+            TermKind::Decl {
```

Comment:
@cfallin: do you think implementing this function (`extractor_sig)` makes sense in terms of returning an `ExternalSig` for an internal extractor? Would it be better to either refactor to separate this function, or rename the return type if we use it for both external and internal?